library/std/src/io/stdio.rs

```diff
@@ -965,3 +965,57 @@ pub fn _eprint(args: fmt::Arguments<'_>) {
 
 #[cfg(test)]
 pub use realstd::io::{_eprint, _print};
+
+/// Trait to determine if stdio stream (Stdin, Stdout, Stderr) is a tty.
+#[unstable(feature = "is_atty", issue = "80937")]
+pub trait IsAtty {
```

Comment:
Slight nit: this is actually 3 words: is a TTY

So the proper name for the trait should be `IsATty`. But that looks pretty ugly so perhaps `IsTerminal` might be a better name.

Reply:
Renamed to IsTerminal and renamed the function and feature accordingly